e2e-tests/development-runtime/package.json

```diff
@@ -40,7 +40,7 @@
     "update:preview": "curl -X POST -d '{ \"fake-data-update\": true }' -H \"Content-Type: application/json\" http://localhost:8000/__refresh",
     "start-server-and-test": "start-server-and-test develop http://localhost:8000 cy:run",
     "cy:open": "cypress open",
-    "cy:run": "(is-ci && cypress run --browser chrome --record) || cypress run --browser chrome"
+    "cy:run": "(is-ci && cypress run --browser chrome --record)"
```

Comment:
did you know that if e2e tests failed in CI that we ran them again because the exit code would be 1 and trigger the || statement 👎

Reply:
ooops, that's on me I think ... goal here was to use `--record` only in CI

Reply:
I think this just won't run locally? so it still need some love?

Reply:
Does anyone really need to run this locally? Usually I use `cy:open` locally

Reply:
so yeah i guess I sort of broke functionality, i can improve if you think it's worth it

Reply:
Can we just drop this change from this PR and handle this separately? It just feels like this doesn't belong in this PR

Reply:
I'll take care of doing this so it doesn't run twice on failures in CI + still runs fine locally